tests/bootstrap.php

```diff
@@ -4,6 +4,6 @@
     throw new RuntimeException('Install dependencies to run test suite.');
 }
 
-$loader = require_once $file;
+$loader = require $file;
```

Comment:
This slipped through when committing - without this I couldn't get my local test suite to run, didn't mean to commit this. Will be removed ;)